content/fr/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,344 @@
+# Utilise aussi "jq"
+for item in $( kubectl get pod --output=name); do printf "Labels for %s\n" "$item" | grep --color -E '[^/]+$' && kubectl get "$item" --output=json | jq -r -S '.metadata.labels | to_entries | .[] | " \(.key)=\(.value)"' 2>/dev/null; printf "\n"; done
+
+# Vérifie quels noeuds sont prêts
+JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}' \
+ && kubectl get nodes -o jsonpath="$JSONPATH" | grep "Ready=True"
+
+# Liste tous les Secrets actuellement utilisés par un pod
+kubectl get pods -o json | jq '.items[].spec.containers[].env[]?.valueFrom.secretKeyRef.name' | grep -v null | sort | uniq
+
+# Liste les événements (Events) classés par timestamp
+kubectl get events --sort-by=.metadata.creationTimestamp
+```
+
+## Mise à jour de ressources
+
+Depuis la version 1.11, `rolling-update` a été déprécié (voir [CHANGELOG-1.11.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.11.md)), utilisez plutôt `rollout`.
+
+```bash
+kubectl set image deployment/frontend www=image:v2               # Rolling update du container "www" du déploiement "frontend", par mise à jour de son image
+kubectl rollout undo deployment/frontend                         # Rollback du déploiement précédent
+kubectl rollout status -w deployment/frontend                    # Écoute (Watch) le status du rolling update du déploiement "frontend" jusqu'à ce qu'il se termine
+
+# déprécié depuis la version 1.11
+kubectl rolling-update frontend-v1 -f frontend-v2.json           # (déprécié) Rolling update des pods de frontend-v1
+kubectl rolling-update frontend-v1 frontend-v2 --image=image:v2  # (déprécié) Modifie le nom de la ressource et met à jour l'image
+kubectl rolling-update frontend --image=image:v2                 # (déprécié) Met à jour l'image du pod du déploiement frontend
+kubectl rolling-update frontend-v1 frontend-v2 --rollback        # (déprécié) Annule (rollback) le rollout en cours
+
+cat pod.json | kubectl replace -f -                              # Remplace un pod, en utilisant un JSON passé en entrée standard
+
+# Remplacement forcé (Force replace), supprime puis re-crée la ressource. Provoque une interruption de service.
+kubectl replace --force -f ./pod.json
+
+# Crée un service pour un nginx repliqué, qui rend le service sur le port 80 et se connecte aux containers sur le port 8000
+kubectl expose rc nginx --port=80 --target-port=8000
+
+# Modifie la version (tag) de l'image du container unique du pod à v4
+kubectl get pod mypod -o yaml | sed 's/\(image: myimage\):.*$/\1:v4/' | kubectl replace -f -
+
+kubectl label pods my-pod new-label=awesome                      # Ajoute un Label
+kubectl annotate pods my-pod icon-url=http://goo.gl/XXBTWq       # Ajoute une annotation
+kubectl autoscale deployment foo --min=2 --max=10                # Mise à l'échelle automatique (Auto scale) d'un déploiement "foo"
+```
+
+## Mise à jour partielle (Patch) de ressources
```

Comment:
as we discussed on slack we all agree to use this formula for first occurence 'french ( english )',
but what about sections titles ? personnaly i'm not sure.